src/scripts/plugindos.js

```diff
@@ -0,0 +1,72 @@
+const registerAdapt = () => {
+    CKEDITOR.plugins.add('libretextsAdapt', {
+      init(editor) {
+        // remove restrictions on what tags and css properties can be output
```

Comment:
I was imagining not needing to to make a second plugin. Can't all the code inside init(editor){} just go in the original plugin.js file? i.e. there is a single plugin that registers two button.

Reply:
Not that we should do this, but a single button could work too. There could be a radio button in the dialog to select "query" or "adapt".

Reply:
Ah I see what you mean, I'll see what I can do about consolidating the code a bit more.

Reply:
Alright, both are now in the same `registerPlugin` function in `plugin.js` for now. I'm not too sure how feasible or possible it is to have both be in `init(editor) {}`, but I'll try to look into reducing redundancy. Having a single button would definitely accomplish that, but I'm unsure of the implications of that from a UI/UX standpoint.

Reply:
Keep it two buttons.

Reply:
Don't fuss over the redundancy too much. Main thing is to get it working.

I was imagining that plugin != button, but maybe that's what plugin sorta means in ckeditor.